### ISSUE TITLE
Apply Woocommerce Table Styling to All `.shop_table` Tables

### ISSUE THREAD

**Issue by thekendog** [soflyy/breakdance-bugs#949]
Currently, certain Woocommerce tables are styled using BD colors.  Here's the current CSS example.

```
.breakdance-woocommerce .woocommerce-grouped-product-list,
.breakdance-woocommerce .woocommerce-product-attributes,
.breakdance-woocommerce table.woocommerce-table--order-details.shop_table,
.breakdance-woocommerce .cart-collaterals table,
.breakdance-woocommerce table.woocommerce-orders-table,
.breakdance-woocommerce table.woocommerce-table--order-downloads,
.breakdance-woocommerce table.woocommerce-table--order-details,
.breakdance-woocommerce table.woocommerce-cart-form__contents,
.breakdance-woocommerce table.woocommerce-checkout-review-order-table
```
This styling doesn't apply to other tables.  For instance, the Woocommerce subscriptions tables that show in my account are not targeted by this.  This styling could be simplified by simply targeting the "shop_table" class.

**Comment by louisreingold** [comment]
i.e. `.breakdance-woocommerce table.shop_table` would get the job done for you?

**Comment by thekendog** [comment]
Yup, that's exactly what I'm looking for.  Would help style tables in a bunch of plugins.

**Comment by breakdancemoderator** [comment]
We tried your suggestion but it turns out not all tables should share the same styles, and applying the styling to `.breakdance-woocommerce table.shop_table` indiscriminately, it will affect tables we don't want.

If you give specific tables that need to be added to the selector, i.e. `.breakdance-woocommerce table.some-specific-table` we're happy to add those.

**Comment by louisreingold** [comment]
@thekendog ^^

**Comment by thekendog** [comment]
The subscriptions one would be good.  Has a class of `woocommerce-orders-table--subscriptions`.

**Comment by breakdancemoderator** [comment]
Thanks for the report. This has been addressed in Breakdance 2.0.
